journal.py

```diff
@@ -117,13 +173,12 @@ def main():
     settings['debug_all'] = debug
     settings['auth.username'] = os.environ.get('AUTH_USERNAME', 'admin')
     manager = BCRYPTPasswordManager()
-    settings['auth.password'] = os.environ.get('AUTH_PASSWORD', manager.encode('secret')
-    )
+    settings['auth.password'] = os.environ.get('AUTH_PASSWORD',
+                                               manager.encode('secret'))
```

Comment:
A bit cleaner looking to break both params onto a single new line indented only four spaces:

```python
settings['foo'] = os.environ.get(
    'KEY', default_val
)
```